konduit-serving-test/src/test/java/ai/konduit/serving/verticles/python/keras/KerasPythonImageJsonFormatTest.java

```diff
@@ -0,0 +1,174 @@
+ *
+ *
+ */
+
+package ai.konduit.serving.verticles.python.keras;
+
+import ai.konduit.serving.InferenceConfiguration;
+import ai.konduit.serving.config.ServingConfig;
+import ai.konduit.serving.miscutils.ExpectedAssertUtil;
+import ai.konduit.serving.miscutils.PythonPathInfo;
+import ai.konduit.serving.model.PythonConfig;
+import ai.konduit.serving.output.types.NDArrayOutput;
+import ai.konduit.serving.pipeline.step.ImageLoadingStep;
+import ai.konduit.serving.pipeline.step.PythonStep;
+import ai.konduit.serving.util.ObjectMappers;
+import ai.konduit.serving.verticles.inference.InferenceVerticle;
+import ai.konduit.serving.verticles.numpy.tensorflow.BaseMultiNumpyVerticalTest;
+import com.jayway.restassured.specification.RequestSpecification;
+import io.vertx.core.AbstractVerticle;
+import io.vertx.core.Handler;
+import io.vertx.core.http.HttpServerRequest;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import io.vertx.ext.unit.TestContext;
+import io.vertx.ext.unit.junit.VertxUnitRunner;
+import org.datavec.python.PythonType;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.nd4j.linalg.api.ndarray.INDArray;
+import org.nd4j.linalg.io.ClassPathResource;
+
+import javax.annotation.concurrent.NotThreadSafe;
+import java.io.File;
+import java.util.Arrays;
+
+import static com.jayway.restassured.RestAssured.given;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+
+@RunWith(VertxUnitRunner.class)
+@NotThreadSafe
+public class KerasPythonImageJsonFormatTest extends BaseMultiNumpyVerticalTest {
+
+    @Override
+    public Class<? extends AbstractVerticle> getVerticalClazz() {
+        return InferenceVerticle.class;
+    }
+
+    @Override
+    public Handler<HttpServerRequest> getRequest() {
+
+        return req -> {
+            //should be json body of classification
+            req.bodyHandler(body -> {
+                System.out.println(body.toJson());
+                System.out.println("Finish body" + body);
+            });
+
+            req.exceptionHandler(exception -> context.fail(exception));
+        };
+    }
+
+    @Override
+    public JsonObject getConfigObject() throws Exception {
+           String pythonCodePath = new ClassPathResource("scripts/keras/KerasImageTest.py").getFile().getAbsolutePath();
+
+        PythonConfig pythonConfig = PythonConfig.builder()
+                .pythonCodePath(pythonCodePath)
+                .pythonPath(PythonPathInfo.getPythonPath())
+                .pythonInput("imgPath", PythonType.TypeName.NDARRAY.name())
+                .pythonOutput("imageArray", PythonType.TypeName.NDARRAY.name())
+                .build();
+
+        PythonStep pythonStepConfig = new PythonStep(pythonConfig);
+
+        //ServingConfig set httpport and Input Formats
+        ServingConfig servingConfig = ServingConfig.builder().httpPort(port).
+                build();
```

Comment:
Better to put `JSON` as output format explicitly.

Reply:
Yes we added explicitly outputformat and committed all the files with similar requirement.